slackbridge/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:wily
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Usual update / upgrade
+RUN apt-get update && apt-get upgrade -y && apt-get dist-upgrade -y
+
+# Install requirements
+RUN apt-get install -y curl supervisor
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_5.x | bash -
+RUN apt-get install --fix-missing -y nodejs
+
+# Get slack-irc
+RUN npm install -g slack-irc
+
+# Add configurations
+ADD config.json slack-irc/config.json
```

Comment:
I agree -- COPY over ADD is definitely a cleaner approach.   I will add that on my next commit.

I thought about using VOLUME when I was modifying this Dockerfile, but I think that adds a layer of complexity to deploying the container.  Since we are only deploying this container in a single location (Heroku) and it's not critical to the CLCP infrastructure, I think it might be okay to keep the configuration 'baked in'.  However, I'll do some local experimentation and see if it helps or hinders our overall approach.  Thanks for the input!